docs/source/sparse.rst

```diff
@@ -1,146 +1,382 @@
-.. currentmodule:: torch.sparse
+.. currentmodule:: torch
 
 .. _sparse-docs:
 
 torch.sparse
 ============
 
-.. warning::
+Torch supports sparse tensors which provide memory-efficient storage
```

Comment:
"Torch" -> "PyTorch"

Reply:
This opening sentence is a little wonky. It has, for example, two "which"s. 

What are the bullet-point goals for this opening paragraph?

Reply:
The bullet-points are:
- What is a sparse tensor? A: It is a tensor with many equal-valued elements.
- Why sparse tensor storage formats are introduced? A: these allow memory-efficient storage for sparse tensors (no need to store the equal-valued elements) as well as processor-efficient algorithms on sparse tensors (no need to process zeros or save in processing of equal-valued elements).
- Don't use a sparse tensor storage format for tensors that are not truly sparse.

Reply:
The paragraph has been rewritten.